controllers/workspace/devworkspace_controller.go

```diff
@@ -309,50 +329,53 @@ func (r *DevWorkspaceReconciler) stopWorkspace(workspace *dw.DevWorkspace, logge
 		Name:      common.DeploymentName(workspace.Status.DevWorkspaceId),
 		Namespace: workspace.Namespace,
 	}
-	status := &currentStatus{}
+	status := initCurrentStatus()
```

Comment:
Well, I think it could simplify the stuff if we start here not from STARTING phase from STOPPING, FAILED, like the following https://github.com/sleshchenko/devworkspace-operator/commit/f7809311360d3ce895d16468600a12f0008284d2

Reply:
`initCurrentStatus` mostly is used to make sure the conditions map is initialized, but I agree that `Starting` isn't ideal. However it should always be set to one of `Stopping/Stopped/Failed` in this function so we should never actually *see* `Starting`

Reply:
let's discuss an alternative proposal in https://github.com/devfile/devworkspace-operator/pull/375#discussion_r614572224